common/Fabric-Install-Utilities.psm1

```diff
@@ -613,7 +613,11 @@ function Write-Console($message){
 }
 
 function Test-DiscoveryHasBuildVersion($discoveryUrl, $credential) {
```

Comment:
this is most useful for running the tests locally?

Reply:
The function is to check to see if the current version of Discovery supports the buildnumber or not. I added the ability to use default credentials for these calls so we don't have to prompt for credentials during our installs.